editcomponents.cgi

```diff
@@ -29,28 +31,31 @@ my $vars     = {};
 # so all actions point to the same page.
 $vars->{'doc_section'} = 'components.html';
 
+print $cgi->header();
+
 #
-# Preliminary checks:
+# Often used variables
 #
+my $user              = Bugzilla->login(LOGIN_REQUIRED);
```

Comment:
All other code calls `Bugzilla->login` before `$cgi->header`.

To avoid surprises please move this before the header is printed.